lib/portlayer/exec/base.go

```diff
@@ -269,12 +269,7 @@ func (c *containerBase) start(op trace.Operation) error {
 		return NotYetExistError{c.ExecConfig.ID}
 	}
 
-	// Power on
-	_, err := c.vm.WaitForResult(op, func(op context.Context) (tasks.Task, error) {
-		return c.vm.PowerOn(op)
-	})
-
-	return err
+	return c.vm.PowerOn(op)
```

Comment:
more of a curiosity. why do we not wait for result here now? is that done upstream from here?

Reply:
It's done behind an abstraction in vm.go.

Reply:
sure i looked through your implementation. Looks good :)